dapr/actor/runtime/config.py

```diff
@@ -42,6 +42,75 @@ def as_dict(self) -> Dict[str, Any]:
         }
 
 
+class ActorTypeConfig:
+    """Per Actor Type Configuration that configures Actor behavior for a specific Actor Type in
+    Dapr Runtime.
+    """
+
+    def __init__(
+            self,
+            actor_type: str,
+            actor_idle_timeout: Optional[timedelta] = None,
+            actor_scan_interval: Optional[timedelta] = None,
+            drain_ongoing_call_timeout: Optional[timedelta] = None,
+            drain_rebalanced_actors: Optional[bool] = None,
+            reentrancy: Optional[ActorReentrancyConfig] = None,
+            reminders_storage_partitions: Optional[int] = None):
+        """Inits :class:`ActorTypeConfig` to configure the behavior of a specific actor type
+        when dapr runtime starts.
+
+        Args:
+            actor_type (str): Actor type.
+            actor_idle_timeout (datetime.timedelta): The timeout before deactivating an idle actor.
+            actor_scan_interval (datetime.timedelta): The duration which specifies how often to scan
+                for actors to deactivate idle actors. Actors that have been idle longer than
+                actor_idle_timeout will be deactivated.
+            drain_ongoing_call_timeout (datetime.timedelta): The duration when in the process of
+                draining rebalanced actors. This specifies the timeout for the current active actor
```

Comment:
```
The duration when in the process of draining rebalanced actors. 
```
Please rephrase

Reply:
Copy pasted the message exactly like this. Will update though